dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jSequenceNode.java

```diff
@@ -149,15 +151,15 @@ private void load() {
 		System.out.println("--- loading sequence node " + getId() + " ---");
 
 		try (Transaction tx = service.beginTx()) {
-			start		= (int)    node.getProperty(PropertyTypes.STARTREF.name());
-			end		= (int)    node.getProperty(PropertyTypes.ENDREF.name());
-			sequence	= (String) node.getProperty(PropertyTypes.SEQUENCE.name());
-			basedist	= (int)    node.getProperty(PropertyTypes.BASE_DIST.name());
-			rank		= (int)    node.getProperty(PropertyTypes.RANK.name());
+			start		= (int) node.getProperty(SequenceProperties.STARTREF.name());
+			end		= (int) node.getProperty(SequenceProperties.ENDREF.name());
+			sequence	= (String) node.getProperty(SequenceProperties.SEQUENCE.name());
+			basedist	= (int) node.getProperty(SequenceProperties.BASE_DIST.name());
+			rank		= (int) node.getProperty(SequenceProperties.RANK.name());
```

Comment:
Please remove the vertical alignment here while we're changing this anyway, or replace with spaces.